external-import/wiz/src/external_import_connector/connector.py

```diff
@@ -147,22 +149,18 @@ def process_message(self) -> None:
             # Store the current timestamp as a last run of the connector
             self.helper.connector_logger.debug(
                 "Getting current state and update it with last run of the connector",
-                {"current_timestamp": current_timestamp},
+                {"current_timestamp": now.timestamp()},
             )
             current_state = self.helper.get_state()
-            current_state_datetime = now.strftime("%Y-%m-%d %H:%M:%S")
-            last_run_datetime = datetime.utcfromtimestamp(current_timestamp).strftime(
-                "%Y-%m-%d %H:%M:%S"
-            )
             if current_state:
-                current_state["last_run"] = current_state_datetime
+                current_state["last_run"] = now
```

Comment:
Your now is currently in datetime format with the tz, but we would like to have an Isoformat + TZ format, here is what I suggest for the current_state :

```suggestion
                current_state["last_run"] = now.isoformat(sep=" ", timespec="seconds")
```